eng/native/signing/auth.json

```diff
@@ -1,18 +1,23 @@
 {
     "Version" : "1.0.0",
     "AuthenticationType" : "AAD_CERT",
-    "TenantId" : "72f988bf-86f1-41af-91ab-2d7cd011db47",
-    "ClientId" : "2234cdec-a13f-4bb2-aa63-04c57fd7a1f9",
+    "TenantId" : "975f013f-7f24-47e8-a7d3-abc4752bf346",
+    "ClientId" : "22346933-af99-4e94-97d5-7fa1dcf4bba6",
     "AuthCert" :
     {
-        "SubjectName" : "CN=2234cdec-a13f-4bb2-aa63-04c57fd7a1f9.microsoft.com",
+        "SubjectName" : "CN=22346933-af99-4e94-97d5-7fa1dcf4bba6.microsoft.com",
         "StoreLocation" : "CurrentUser",
         "StoreName": "My",
-        "SendX5c" : "true"
+        "SendX5c" : "true",
+        "WithAzureRegion": false
     },
     "RequestSigningCert" : {
-        "SubjectName" : "CN=2234cdec-a13f-4bb2-aa63-04c57fd7a1f9",
-        "StoreLocation" : "CurrentUser",
-        "StoreName" : "My"
-    }
+        "GetCertFromKeyVault" : true,
```

Comment:
We can't fetch the other cert from key vault, right?

Reply:
No, this one always needs to be installed if not using an MI. And we can't really use one here as things stand today.